packages/Organisms/Markup/examplePlugins.js

```diff
@@ -61,37 +64,105 @@ const pluginEmbeddedDocument = function () {
             case 'file--mime-text-csv':
               fileType = 'csv'
               break
+            case 'file--mime-text-calendar':
+              fileType = 'ics'
+              break
           }
         }
       })
     }
 
+    if (url) {
+      url = url.replace(/^.*\/\/[^/]+/, '')
+    }
+
     if (fileType === '') {
-      fileType = $root.find('.file--type').text().toLowerCase()
+      fileType = el.find('.file--type').text().toLowerCase()
     }
 
     if (url && fileName && fileSize && fileType) {
       const documentlink = `<rpl-document-link name="${fileName}" extension="${fileType}" filesize="${fileSize}" url="${url}" caption="${caption}"></rpl-document-link>`
-      return $root.replaceWith(documentlink)
+      return el.replaceWith(documentlink)
     }
-    return $root
+    return el
+  })
+}
+
+const parseForLinks = function () {
+  // Give h2 headings an id so they can be linked to
+  const kebabCase = require('lodash.kebabcase')
+
+  this.find('h2').map((i, element) => {
+    const el = this.find(element)
+    const idName = el.text()
+    return el.attr('id', kebabCase(idName))
+  })
+}
+
+const pluginIframe = function () {
+  // wrap iFrames
+  const wrapperClasses = ['rpl-markup__iframe-container', 'rpl-markup__iframe-container--16x9']
+  this.find('iframe').map((i, el) => {
+    const iframe = this.find(el)
+    const markup = `<div class="${wrapperClasses.join(' ')}"></div>`
+    return iframe.wrap(markup)
+  })
+}
+
+const pluginEmbeddedMediaVideo = function () {
+  // wrap iFrames
+  this.find('.embedded-entity--media--embedded-video').map((i, el) => {
+    const element = this.find(el)
+    const iframe = this.find('iframe')
+    const height = iframe.attr('height')
+    const width = iframe.attr('width')
+    const src = iframe.attr('src')
+    const lang = iframe.attr('lang')
+    const figcaption = this.find('figcaption')
+    const transcript = figcaption ? figcaption.text() : null
+    const link = this.find('.field--name-field-media-link a')
+    const mediaLink = link ? `{ text: '${link.text()}', url: '${link.attr('href')}' }` : null
+    const RplEmbeddedVideo = `
+      <rpl-embedded-video
+        width="${width}"
+        height="${height}"
+        src="${src}"
+        lang="${lang}"
+        variant="link"
+        ${link && ':media-link="' + mediaLink + '"'}
+        ${transcript && 'transcript="' + transcript + '"'}
+      />
+    `
+    return element.replaceWith(RplEmbeddedVideo)
   })
 }
 
 const pluginLinks = function () {
   this.find('a').map((i, el) => {
     const $a = this.find(el)
     const href = $a.attr('href')
+    // const text = htmlUtilities.decodeSpecialCharacters($a.text())
```

Comment:
I'm guessing this is a nuxt-specific function that we can't use in ripple at the moment. It's good to leave some sort of notification in there that "you'll likely need to decode your html here" - so I can't decide if this comment should stay or go.